python-lib/design_experiment/ab_dispatcher.py

```diff
@@ -1,42 +1,41 @@
 import numpy as np
+from numpy.random import Generator
```

Comment:
not used anymore

Reply:
done in https://github.com/dataiku/dss-plugin-ab-testing/pull/1/commits/244d99c0ead1707a487370eb305d4d158c84836e